Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Cursors/BaseCursor.cs

```diff
@@ -82,11 +83,21 @@ public class BaseCursor : MonoBehaviour, IMixedRealityCursor
         private Vector3 targetScale;
         private Quaternion targetRotation;
 
-        #region IMixedRealityCursor Implementation
-
         /// <summary>
-        /// The pointer that this cursor should follow and process input from.
+        /// Indicates if the cursor should be visible
         /// </summary>
+        public bool IsVisible
+        {
+            set
```

Comment:
Should have a getter also...

Reply:
to me, a set only property is bad form

Reply:
Isn't this the same in master?
But yea I agree we should be able to get as well.

Reply:
to be honest, I didn't check :)

if we want to take this and file an issue to fix, I would be ok with that.